### PR TITLE
Fix Feedback Time Errors

### PR DESCRIPTION
**Pull request by brianlugo1**
This pull request fixes:
 - Errors with the /rating GET route to return the user and completed assessment for a given assessment task!
 - Errors with the deleting of ratings.py and the updated schemas!
 - Also fixes the demo data to have something to display when Reporting!

### CODE REVIEW

**Review by Wrimo — CHANGES_REQUESTED** [review]
From my understanding of Feedback Time, this does yet accomplish the goal. I believe the sequence of events is as follows:

(1) Assessment Task is completed 
 |
V
(2) Admin/Instructor reviews and clicks to alert students of feedback
 |
V
(3) Students views feedback 
 |
V
(4) Admin can view ratings and see student lag time. 

The lag time is supposed to be the time between (2) and (3). As currently implemented, it's the time between (2) and (4)*. Part of the problem is that I don't think there's any mechanism for (3) to happen yet. In order for this to be completed correctly, there will need a route for (3) that stores the time the student views their feedback and a separate route for (4) to get the ratings. Currently. they are the same route. 

\* Technically its between whenever the feedback is created and (4). I believe its intended to happen at (2), there's just no code in the front end yet to actually call the feedback creation route.

**Comment by Wrimo** [comment]
Alright, I just made a push to this branch that should fix the issues I discussed in my previous comment. 

When a student views their feedback/completed assessment task, the front end needs to call a new route called `student_view_feedback` which creates the database entry for that student on that completed assessment task and stores the time of creation as the view time. 

Then when an admin views the ratings and the other rating route, `get_student_individual_ratings`, is called, the lag time is computed and returned with the rest of the time. This route is currently only set up to work with individual assessment tasks, but I don't think it would require much modification to get it to work for team assessment tasks as well.

File: BackEndFlask/models/queries.py [hunk 1]
```diff
@@ -157,10 +171,18 @@ def get_users_by_team_id(team):
         ).all()
 
     except SQLAlchemyError as e:
-        error = e.__dict__['orig']
-        return error
+        logger.error(str(e.__dict__['orig']))
+        raise e
 
 def get_users_not_in_team_id(team):
+    """
```
Comment (malloc-nbytes) [inline BackEndFlask/models/queries.py:178]: this has the same description that `get_users_by_team_id()` has.

**Review by malloc-nbytes — APPROVED** [review]
Other than the comment, LGTM

File: BackEndFlask/models/queries.py (hunk 1 above)
Comment (brianlugo1) [reply BackEndFlask/models/queries.py:176]: @malloc-nbytes I have updated the descriptions!

**Comment by brianlugo1** [comment]
@sah0017 I am going to go ahead and merge this PR with only one approved code review!

**Pull request merged**